src/commands/deploy/deploy.ts

```diff
@@ -81,6 +81,14 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
         return await getOrCreateFunctionApp(context)
     });
 
+    if (node.contextValue.includes('container')) {
+        const learnMoreLink: string = 'https://aka.ms/deployContainerApps'
+        await context.ui.showWarningMessage(localize('containerFunctionAppError', 'Deploy is not supported for containerized function apps within the Azure Functions extension. Please use the Azure Container Apps extension to deploy your project.'), { learnMoreLink });
+        //suppress display of error message
+        context.errorHandling.suppressDisplay = true;
+        throw new Error();
```

Comment:
We may want to make a new error for telemetry purposes. So we can track when the command is being used to deploy to a container app.